controllers/utils/constants.go

```diff
@@ -2,6 +2,13 @@ package utils
 
 import "k8s.io/apimachinery/pkg/runtime/schema"
 
+// DefaultCGUControllerWorkerCount default number of worker threads for the CGU controller
+// CGUController constants
+const (
+	CGUCtrllWorkerCountEnv          = "TALM_CGU_CTRL_WORKER_COUNT"
```

Comment:
To be consistent, either refer to both CGU controller constants as `...CGUControllerWorker...` or `...CGUCtrlWorker...`